docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,87 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
+
+1. Generate AWS Long Term Access Key
```

Comment:
Remove this part. We'll assume they have access.

Reply:
@laurenfrederick This is removed.